### DISCUSSION TITLE
Is there a way to disable ObjectSpace in jirb_swing?

### DISCUSSION THREAD

**Discussion by wrq** [jruby/jruby#8150] [Q&A] [answered]
When I try to invoke ``jirb_swing.bat -O`` I get an ``Unrecognized switch: -O`` error from the IRB module. 

(Latest windows, latest Jruby installed via Windows installer)

Is there an easy fix that I am overlooking? Should this be supported in some way? 

Thanks for reading.

**Comment by headius** [comment, marked as answer]
There is no way, but it's something that could (should?) be added. Perhaps you could look at bin/jirb_swing and see if you can add an option there?

**Reply by wrq** [reply]
I will take a peek when I can and see
